autoload/smoothie.vim

```diff
@@ -6,6 +6,12 @@
 " invokes motion (except smoothie#forwards, where it must be set to v:true)
 let s:ctrl_f_invoked = v:false
 
+if !exists('g:smoothie_disabled')
```

Comment:
Most plugins I've seen (f.ex. airblade/vim-gitgutter, Yggdroot/indentLine, unblevable/quick-scope) have a `g:<plugin_name>_enabled` switch. I think we should follow similar convention here for a more consistent user experience.